build-setup-nolog.sh

```diff
@@ -64,6 +69,21 @@ do
     shift
 done
 
+# before doing anything verify that you are on a release branch/tag
+set +e
+tag=$(git describe --exact-match --tags)
+tag_ret_code=$?
+set -e
+if [ $tag_ret_code -ne 0 ]; then
+    if [ "$SKIP_VALIDATE" = false ]; then
+        read -p "WARNING: You are not on a tagged release of FireSim. Type \"ok\" to continue: " validate
```

Comment:
"on a tagged" -> "on an official"

Also, I would replace "Type \"ok..." with:

"\n Type \"ok\" to continue if this is intended, otherwise see [add link to the part of the setting up manager page with the clone instruction]"